<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlueApplication</class>
 <widget class="QMainWindow" name="GlueApplication">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Glue</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="LayerTreeWidget" name="layerWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Tab 1</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Tab 2</string>
        </attribute>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1120</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_session"/>
    <addaction name="actionNew_Session"/>
    <addaction name="actionQuit"/>
    <addaction name="actionNew_Tab"/>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="actionLoad_data"/>
   </widget>
   <widget class="QMenu" name="menuSubsets">
    <property name="title">
     <string>Subsets</string>
    </property>
    <addaction name="actionNew_Subset"/>
    <addaction name="actionCombine_Subsets"/>
    <addaction name="actionSave_Subset"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuPanel_Grid">
     <property name="title">
      <string>Panel Grid</string>
     </property>
     <addaction name="action1x1"/>
     <addaction name="action2x2"/>
     <addaction name="action2x2_2"/>
    </widget>
    <addaction name="actionNew_Tab_2"/>
    <addaction name="menuPanel_Grid"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuData"/>
   <addaction name="menuSubsets"/>
   <addaction name="menuHelp"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave_session">
   <property name="text">
    <string>Save session</string>
   </property>
  </action>
  <action name="actionNew_Session">
   <property name="text">
    <string>New session</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionLoad_data">
   <property name="text">
    <string>Load data</string>
   </property>
  </action>
  <action name="actionNew_Subset">
   <property name="text">
    <string>New Subset</string>
   </property>
  </action>
  <action name="actionCombine_Subsets">
   <property name="text">
    <string>Combine Subsets</string>
   </property>
  </action>
  <action name="actionSave_Subset">
   <property name="text">
    <string>Save Subset</string>
   </property>
  </action>
  <action name="actionNew_Tab">
   <property name="text">
    <string>New Tab</string>
   </property>
  </action>
  <action name="actionNew_Tab_2">
   <property name="text">
    <string>New Tab</string>
   </property>
  </action>
  <action name="action1x1">
   <property name="text">
    <string>1x1</string>
   </property>
  </action>
  <action name="action2x2">
   <property name="text">
    <string>2x2</string>
   </property>
  </action>
  <action name="action2x2_2">
   <property name="text">
    <string>3x3</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LayerTreeWidget</class>
   <extends>QWidget</extends>
   <header>layertreewidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
